test/verify/netlib.py

```diff
@@ -116,3 +116,13 @@ def slow_down_dhclient(self, delay):
         m.execute("mv /usr/sbin/dhclient /usr/sbin/dhclient.real")
         m.write("/usr/sbin/dhclient", '#! /bin/sh\nsleep %s\nexec /usr/sbin/dhclient.real "$@"' % delay)
         m.execute("chmod a+x /usr/sbin/dhclient")
+
+    def wait_onoff(self, sel, val):
+        self.browser.wait_present(sel + " .btn.active:contains('%s')" % ("On" if val else "Off"))
+
+    def toggle_onoff(self, sel):
+        # PR #11773 moves to our React component, which changes what we have to click on
+        if self.machine.image in ["rhel-7-6-distropkg", "rhel-8-0-distropkg"]:
+            self.browser.click(sel + " .btn:contains('On')")
+        else:
+            self.browser.click(sel + " .btn:contains('On') span")
```

Comment:
The extra "span" doesn't hurt the old tests, so if always clicking on "on" actually works, this special case can just go away completely. I just thought that for the React component you always have to click on the non-current half of it, but maybe/hopefully that was wrong then.

Reply:
Let's try that!